fr/chronos.rst

```diff
@@ -0,0 +1,237 @@
+
+* Les objets ``Date`` pour représenter les dates du calendrier.
+* Les objets pour les dates immutables et les datetimes.
+* Un système de traduction intégrable. Seules les traductions anglaises sont
+  inclues dans la librairie. Cependant, ``cakephp/i18n`` peut être utilisé
+  pour un support complet d'autres langues.
+
+Installation
+============
+
+Pour installer chronos, vous devez utiliser ``composer``. A partir du répertoire
+ROOT de votre application (où le fichier composer.json se trouve) exécutez ce
+qui suit::
+
+    php composer.phar require cakephp/chronos "@stable"
+
+Vue d'Ensemble
+==============
+
+Chronos fournit un certain nombre d'extensions pour les objets DateTime fournis
+par PHP. Chronos fournit 5 classes qui gèrent les variantes mutables et
+immutables de date/time et les extensions pour ``DateInterval``.
+
+* ``Cake\Chronos\Chronos`` est un objet de *date et time* immutable.
+* ``Cake\Chronos\Date`` est un objet de *date* immutable.
+* ``Cake\Chronos\MutableTime`` est un objet de *date et time* mutable.
+* ``Cake\Chronos\MutableDate`` est un objet de *date* mutable.
+* ``Cake\Chronos\ChronosInterval`` est une extension pour l'objet
+  ``DateInterval``.
+
+Enfin si vous voulez typer selon les objets date/time fournis par chronos,
+vous devez utiliser ``Cake\Chronos\ChronosInterface``. Tous les objets date et
+time implémentent cette interface.
+
+Créer des Instances
+===================
+
+Il y a plusieurs façons d'obtenir une instance de Chronos ou de Date. Il y a
+un certain nombre de méthodes factory qui fonctionnent avec des ensembles
+ayant des arguments différents::
+
+    use Cake\Chronos\Chronos;
+
+    $now = Chronos::now();
+    $today = Chronos::today();
+    $yesterday = Chronos::yesterday();
+    $tomorrow = Chronos::tomorrow();
+
+    // Parse les expressions relatives
+    $date = Chronos::parse('+2 days, +3 hours');
+
+    // Les valeurs d'entier de Date et time.
+    $date = Chronos::create(2015, 12, 25, 4, 32, 58);
+
+    // Les valeurs d'entier pour Date ou time.
+    $date = Chronos::createFromDate(2015, 12, 25);
+    $date = Chronos::createFromTime(11, 45, 10);
+
+    // Parse les valeurs formatés.
+    $date = Chronos::createFromFormat('m/d/Y', '06/15/2015');
+
+Travaille avec les Objets Immutables
+====================================
+
+Si vous avez utilisé les objets ``DateTime`` de PHP, vous êtes à l'aise avec
+les objets *mutable*. Chronos offre des objets mutables, mais elle fournit aussi
+des objets *immutables*. Les objets Immutables créent des copies des objets à
+chaque fois qu'un objet est modifié. Puisque les méthodes de modification
+autour des datetimes ne sont pas toujours transparentes, les données peuvent
+être modifiées accidentellemnt ou sans que le développeur ne le sache. Les
+objets immutables évitent les changements accidentels des données et
+make code free of order based dependency issues. L'immutabilité signifie que
```

Comment:
@HavokInspiration reading the code example  in the folowing paragraph will probably help you to better understand. 
https://github.com/cakephp/docs/blob/3.2/en/chronos.rst#working-with-immutable-objects